app/backend/src/couchers/models/background_jobs.py

```diff
@@ -52,7 +52,7 @@ class BackgroundJob(Base):
     payload: Mapped[bytes] = mapped_column(Binary)
 
     # if the job failed, we write that info here
-    failure_info: Mapped[str | None] = mapped_column(String, nullable=True)
```

Comment:
Confirming, this is not needed because of https://docs.sqlalchemy.org/en/21/orm/mapping_api.html#sqlalchemy.orm.mapped_column ?

> If omitted, the nullability is derived from the type annotation based on whether or not typing.Optional (or its equivalent) is present.